thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -212,39 +189,20 @@ export function filterPrefix(urns, prefixes) {
   return makeIterable(urns).filter(urn => hasPrefix(urn, prefixes));
 }
 
-export function toBaselineRange(currentRange, compareMode) {
+export function toBaselineRange(anomalyRange, compareMode) {
   const offset = {
-    current: 0,
-    wow: 1,
-    wo1w: 1,
-    wo2w: 2,
-    wo3w: 3,
-    wo4w: 4
-  }[compareMode.toLowerCase()];
-
-  if (offset === 0) {
-    return currentRange;
-  }
+    WoW: 1,
+    Wo2W: 2,
+    Wo3W: 3,
+    Wo4W: 4
+  }[compareMode];
```

Comment:
Blocking comment until we determine if this breaks rca/rootcause: This `toBaselineRange` util function is being shared with 4 services and 1 components. I feel that changing [compareMode.toLowerCase()] to  [compareMode] would introduce breakage. @apucher can you confirm if this is ok.